src/System.Slices/src/System.Slices.csproj

```diff
@@ -1,55 +0,0 @@
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <OutputType>Library</OutputType>
-    <RootNamespace>System.Slices</RootNamespace>
-    <AssemblyName>System.Slices</AssemblyName>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <ProjectGuid>{C5FD8740-19EA-4BCC-B518-7F16B55D23CA}</ProjectGuid>
-  </PropertyGroup>
-  <!-- Default configurations to help VS understand the configurations -->
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-  </PropertyGroup>
-  <ItemGroup>
-    <None Include="project.json" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="System\Contract.cs" />
-    <Compile Include="System\MemoryUtils.cs" />
-    <Compile Include="System\PrimitiveAttribute.cs" />
-    <Compile Include="System\PtrUtils.cs" />
-    <Compile Include="System\Span.cs" />
-    <Compile Include="System\SpanEnumerators.cs" />
-    <Compile Include="System\SpanExtensions.cs" />
-    <Compile Include="System\SpanEqualsExtensions.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>SpanExtensionsTemplate.tt</DependentUpon>
-    </Compile>
-    <Compile Include="System\SpanHelpers.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Content Include="System\SpanExtensionsTemplate.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>SpanEqualsExtensions.cs</LastGenOutput>
-    </Content>
-  </ItemGroup>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-  <PropertyGroup>
-    <ILDasm Condition="'$(ILDasm)' == '' AND Exists('ildasm.exe')">ildasm.exe</ILDasm>
-    <ILDasm Condition="'$(ILDasm)' == '' AND Exists('$(MSBuildProgramFiles32)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6 Tools\ildasm.exe')">$(MSBuildProgramFiles32)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6 Tools\ildasm.exe</ILDasm>
-    <ILAsm Condition="'$(ILAsm)' == '' AND Exists('ilasm.exe')">ilasm.exe</ILAsm>
-    <ILAsm Condition="'$(ILAsm)' == '' AND Exists('$(WinDir)\Microsoft.NET\Framework\v4.0.30319\ilasm.exe')">$(WinDir)\Microsoft.NET\Framework\v4.0.30319\ilasm.exe</ILAsm>
-    <DecompileToILCommand>"$(ILDasm)" /linenum /out:"$(TargetDir)$(TargetName).beforerewrite.il" /nobar "$(TargetPath)"</DecompileToILCommand>
-    <RewriteILCommand>"$([System.IO.Path]::GetFullPath('$(SourceDir)System.Slices\tools\ILSub\ILSub.exe'))" "$(TargetDir)$(TargetName).beforerewrite.il" "$(TargetDir)$(TargetName).rewritten.il"</RewriteILCommand>
-    <RecompileILCommand>"$(ILAsm)" /pdb /quiet /dll /out:$(TargetPath) /nologo $(TargetDir)$(TargetName).rewritten.il</RecompileILCommand>
-    <PostBuildEvent Condition="'$(ILDasm)' != '' AND '$(ILAsm)' != ''">$(DecompileToILCommand) &amp;&amp; $(RewriteILCommand) &amp;&amp; $(RecompileILCommand)</PostBuildEvent>
```

Comment:
Ok. I ended up just adding it back to the xproj file, instead of the project.json file. I am not sure how to figure out output path and other things in the project.json (or a script invoked for it). This is a very simple fix and corefxlab is still using msbuild as its driver, even though it uses dotnet cli for build/compilation.